operations/ls.go

```diff
@@ -75,31 +76,27 @@ func (so *SingleOperator) listRecursively(
 	}
 }
 
-func splitPattern(path string) (base, pattern string) {
-	hasMeta := false
-
-	splitIdx := -1
-	for i := 0; i < len(path); i++ {
-		c := path[i]
-		if c == '\\' {
-			i++
-		} else if c == '/' {
-			splitIdx = i
-		} else if c == '*' || c == '?' || c == '[' || c == '{' {
-			hasMeta = true
-			break
-		}
+func hasMeta(path string) bool {
```

Comment:
Maybe we can remove this function? if `doublestar.SplitPattern(path) == path`, the path must not have meta?

Reply:
In the process of expanding the path step by step, it may not contain meta at a certain level, like `test/a*/b/c*`?